server/svix-server/migrations/20231010170520_messagecontent_table.up.sql

```diff
@@ -0,0 +1,11 @@
+CREATE TABLE messagecontent (
+    id character varying NOT NULL COLLATE pg_catalog."C",
```

Comment:
Does this share an id with the corresponding message? I sort of expected a nullable fk added on the message side, but I suppose that's not needed if the ids just match.

Reply:
Yeah, the IDs match. Seemed simple to me.